src/dataaccess/mcodev0.1-datasource/model/procedure/FluxProcedurePerformed.js

```diff
@@ -0,0 +1,46 @@
+import TimePeriod from '../shr/core/TimePeriod';
+import ProcedurePerformed from '../shr/procedure/ProcedurePerformed';
+
+class FluxProcedurePerformed {
+    constructor(json) {
+        this._procedurePerformed = ProcedurePerformed.fromJSON(json);
+    }
+
```

Comment:
This class (and the other `FluxProcedurePerformed` class in the v0.1 class structure) needs an `entryInfo` getter that gets the entryInfo from the contained `_procedurePerformed` , otherwise it can crash the reference lookup in PatientRecord.getEntryFromReference. At a glance I don't see any others where it's missing but someone please double check me on that.

Reply:
Thanks, I pushed up the getters.  I don't think we have any instances of `ProcedurePerformed` so didn't catch this in our endpoints.